package.json

```diff
@@ -32,6 +32,7 @@
 		"webpack-dev-server": "^3.1.14"
 	},
 	"dependencies": {
-		"jquery": "^3.4.1"
+		"jquery": "^3.4.1",
+		"moment": "^2.29.1"
```

Comment:
not quite sure what "moment" is but Im sure this is tracking the current date

Reply:
^ ditto.

Reply:
`moment` is a package that helps format the current date.